CollAction/Controllers/ProjectsController.cs

```diff
@@ -161,9 +161,15 @@ public async Task<IActionResult> Create(CreateProjectViewModel model)
             foreach (var admin in administrators)
                 _emailSender.SendEmail(admin.Email, subject, confirmationEmailAdmin);
 
-            return View("ThankYouCreate", new ThankYouCreateProjectViewModel()
+            return LocalRedirect($"~/Projects/Create/{Uri.EscapeDataString(project.Name.NormalizeUriPart())}/{project.Id}/thankyou");
```

Comment:
If at the end of NormalizeUri you:

- Strip any character that has to be encoded (Anything not ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789-._~!'()*)
- Then, guard for empty result strings, and replace those with a single dash

Then you don't need a second encode. Any time you start encoding things (emoticons, or whatever's leftover), the url will look ugly. We can talk through skype / slack if you want to figure out some options.

Reply:
Fixed in c519dac